src/Admin/AbstractAdmin.php

```diff
@@ -2050,6 +2052,22 @@ final protected function getDefaultFilterValues(): array
         return $defaultFilterValues;
     }
 
+    /**
+     * @return array<string, mixed>
+     */
+    final protected function getFormOptions(): array
```

Comment:
could this be private?

Reply:
IMO better will be keep it protected. You can override `AbstractAdmin::getFormBuilder()` to add default form options where visibility of this method is required.